src/sparkx/Lattice3D.py

```diff
@@ -1004,8 +1004,14 @@ def add_particle_data(self, particle_data, sigma, quantity, add = False):
             else:
```

Comment:
We should add strangeness here as well. The particle class has the `is_strange()` function.